test/unit/uk/gov/hmrc/organisationsdetailsapi/domain/numberofemployees/NumberOfEmployeeCountsSpec.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package unit.uk.gov.hmrc.organisationsdetailsapi.domain.numberofemployees
+
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import play.api.libs.json.Json
+import uk.gov.hmrc.organisationsdetailsapi.domain.numberofemployees.NumberOfEmployeeCounts
+
+class NumberOfEmployeeCountsSpec extends AnyWordSpec with Matchers {
+  "Writes to json successfully" in {
+    val expectedJson =
+      """
+        |{
+        |   "numberOfEmployees": "1234",
```

Comment:
more of a question than anything - when do we want to wrap numbers in quotes and when do we want them as numbers?  since here : resources/public/api/conf/1.0/examples/number-of-employees-response.json numberOfEmployees is a number

Reply:
Good spot @Luka-Abey . They would only want to be in quotes if it is a string. I think the quotes need to be removed here?